schema.sql

```diff
@@ -0,0 +1,15 @@
+-- Member 1 Work
+
+CREATE TABLE LABEL(
+  ID INT PRIMARY KEY GENERATED ALWAYS AS IDENTITY,
+  TITLE VARCHAR,
+  COLOR VARCHAR
+);
+
+CREATE TABLE BOOKS(
+  ID INT PRIMARY KEY GENERATED ALWAYS AS IDENTITY,
+  PUBLISHER VARCHAR,
+  COVER_STATE VARCHAR,
+  PUBLISH_DATE DATE,
+  ARCHIVED BOOLEAN,
+);
```

Comment:
Good work on creating the `books` table, but the requirements mentioned: 
> books table (add all properties and associations from the parent Item class as table columns)
so I think we have to add all the properties of the `item` class

Reply:
@teckim No they mean the Properties of book class and ALL other properties you are inheriting as super or calling it in INITIALIZATION.